roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -107,30 +107,27 @@ def get_var(self, *keys, **kwargs):
                 return kwargs["default"]
             raise OpenShiftCheckException("'{}' is undefined".format(".".join(map(str, keys))))
 
-        convert = kwargs.get("type")
+        convert = kwargs.get("convert", lambda x: x)
         try:
-            if convert is None:
-                pass
-            elif convert in ["str", str]:
-                return str(value)
-            elif convert in ["int", int]:
-                return int(value)
-            elif convert in ["float", float]:
-                return float(value)
-            elif convert in ["bool", bool]:
+            if convert is bool:
```

Comment:
Suggestion: instead of changing what `bool` (the function) means behind the scenes, we could have an explicit conversion function that handles all the corner cases we want, that would be similar to how API like Django Models and other Python libraries work.

We'd essentially declare `type=SpecialBool` (surely name `SpecialBool` anything different than that...) for the variables where we want the special handling. That also leaves room for us to use `bool` in the future without breaking any then existing code.